app/uk/gov/hmrc/cataloguefrontend/MetricsExplorerController.scala

```diff
@@ -38,9 +38,10 @@ class MetricsExplorerController @Inject()(
   def landing: Action[AnyContent] =
     Action.async { implicit request =>
       for {
-        groups <- metricsConnector.getAllGroups
-        dependencies <- metricsConnector.getAllDependencies
-        repositories <- metricsConnector.getAllRepositories
+        allMetricsData <- metricsConnector.allMetricsData
```

Comment:
I understand the change, but I can't see anything changed in unit tests. Were there no changes to the tests required?